src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -8488,6 +8489,76 @@ class StubGenerator: public StubCodeGenerator {
     if (UseAdler32Intrinsics) {
       StubRoutines::_updateBytesAdler32 = generate_updateBytesAdler32();
     }
+
+#ifdef COMPILER2
+    // Get sleef stub routine addresses
+    char ebuf[1024];
+    void* libsleef = os::dll_load(UseSleefLib, ebuf, sizeof ebuf);
```

Comment:
Shouldn't this check that UseSleefLib has been set to something other than "" ? (To save the failing `dll_load` call.)

Reply:
Yeah, it's better to do that. Currently it returns "nullptr" without any errors. But I agree that having a pre-check is better. Thanks!

Reply:
`UseSleefLib` is removed in latest commit. So this change is not needed anymore. Thanks for the suggestion all the time!